### ISSUE TITLE
VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE doesn't do anything

### ISSUE THREAD

**Issue by pm7y** [super-linter/super-linter#6394]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Super-linter version available?

- [X] I am using the latest Super-linter version.
- [X] I can reproduce the issue running Super-linter using complete version identifier (example: vX.Y.Z), and not just with a partial one (example: vX)

- [X] I am using the super-linter/super-linter action or container image, and not the deprecated github/super-linter action or container image.

### Are you resonably sure that it's a Super-linter issue, and not an issue related to a tool that Super-linter runs?

- [X] I think that this is a Super-linter issue.

### Current Behavior

Passing `VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE: false` and still whitespace validation errors are being raised in C# dotnet files. e.g. "error WHITESPACE: Fix whitespace formatting. Delete 4 characters."

### Expected Behavior

When passing `VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE: false` the expected behaviour is that whitespace issues in C# dotnet file will not be validated and errors will not be raised.

### Super-Linter version

```markdown
v6.9.0
```

### Super-linter configuration

```yaml
- name: Super Linter
        uses: super-linter/super-linter@v6
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE: false
```

### Relevant log output

```shell
------
  2024-11-26 07:36:00 [INFO]   Stderr contents for CSHARP:
  ------
  /github/workspace/src/XXX/v1/YYY/Validator.cs(17,9): error WHITESPACE: Fix whitespace formatting. Delete 4 characters. [/github/workspace/src/XXX/v1/YYY/]
  /github/workspace/src/XXX/v1/YYY/Validator.cs(18,13): error WHITESPACE: Fix whitespace formatting. Delete 4 characters. [/github/workspace/src/XXX/v1/YYY/]
  /github/workspace/src/XXX/v1/YYY/Validator.cs(19,13): error WHITESPACE: Fix whitespace formatting. Delete 4 characters. [/github/workspace/src/XXX/v1/YYY/]
  /github/workspace/src/XXX/v1/YYY/Validator.cs(20,13): error WHITESPACE: Fix whitespace formatting. Delete 4 characters. [/github/workspace/src/XXX/v1/YYY/]
  /github/workspace/src/XXX/v1/YYY/Validator.cs(21,9): error WHITESPACE: Fix whitespace formatting. Delete 4 characters. [/github/workspace/src/XXX/v1/YYY/]
  ------
GITLEAKS
------
2024-11-26 07:36:00 [INFO]   ----------------------------------------------
2024-11-26 07:36:00 [INFO]   ----------------------------------------------
Error: -26 07:36:01 [ERROR]   Errors found in CSHARP
Notice: 26 07:36:01 [NOTICE]   Successfully linted GITLEAKS
Error: -26 07:36:01 [ERROR]   Super-linter detected linting errors
```

### Steps To Reproduce

1. Create a whitespace issue in a c# file
2. Pass VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE: false
3. Still get error

**Comment by ferrarimarco** [comment]
Hi, thanks for reporting this issue.

Super-linter is working as expected here. To disable that check, you should set `VALIDATE_CSHARP` to `false`. If you have `.sln` files for solutions that include those `.cs` files, you also need to set `VALIDATE_DOTNET_SLN_FORMAT_WHITESPACE` to `false` as you're already doing.

`CSHARP` looks for `.cs` files, while `SLN` looks for `.sln` files.

**Comment by ferrarimarco** [comment]
`VALIDATE_CSHARP`: run

https://github.com/super-linter/super-linter/blob/6b60f4c968c6ef706f82d77c242d2e6d398868ff/lib/functions/linterCommands.sh#L129

against `.cs` files

`VALIDATE_DOTNET_SLN_...`: run 

https://github.com/super-linter/super-linter/blob/6b60f4c968c6ef706f82d77c242d2e6d398868ff/lib/functions/linterCommands.sh#L134-L136

against `.sln` files, and all files that these `.sln` files reference

`VALIDATE_CSHARP` was implemented in the super-linter early days.
